server/src/main/java/org/eclipse/openvsx/publish/PublishExtensionVersionHandler.java

```diff
@@ -53,22 +60,23 @@ public class PublishExtensionVersionHandler {
     ExtensionValidator validator;
 
     @Transactional
-    public ExtensionVersion createExtensionVersion(ExtensionProcessor processor, PersonalAccessToken token) {
+    public ExtensionVersion createExtensionVersion(ExtensionProcessor processor, PersonalAccessToken token, LocalDateTime mirror) {
         // Extract extension metadata from its manifest
-        var extVersion = createExtensionVersion(processor, token.getUser(), token);
-        var dependencies = processor.getExtensionDependencies().stream()
-                .map(this::checkDependency)
-                .collect(Collectors.toList());
-        var bundledExtensions = processor.getBundledExtensions().stream()
-                .map(this::checkBundledExtension)
-                .collect(Collectors.toList());
-
-        extVersion.setDependencies(dependencies);
-        extVersion.setBundledExtensions(bundledExtensions);
+        var extVersion = createExtensionVersion(processor, token.getUser(), token, mirror);
+        var dependencies = processor.getExtensionDependencies().stream();
+        var bundledExtensions = processor.getBundledExtensions().stream();
+        if (mirror == null) {
```

Comment:
Why skip checking dependencies in mirror mode?

Reply:
- It is the mirror mode, checks are done upstream.